WordPress/src/main/java/org/wordpress/android/ui/quickstart/QuickStartFullScreenDialogFragment.kt

```diff
@@ -72,7 +82,17 @@ class QuickStartFullScreenDialogFragment : Fragment(R.layout.quick_start_dialog_
         val selectedSiteLocalId = selectedSiteRepository.getSelectedSiteLocalId().toLong()
         val tasksCompleted = quickStartStore.getCompletedTasksByType(selectedSiteLocalId, tasksType)
         val taskCards = tasks.mapToQuickStartTaskCard(tasksCompleted)
-        quickStartAdapter.submitList(taskCards)
+        val headerTitleResId = quickStartCardBuilder.getTitle(tasksType)
+        val quickStartList = mutableListOf<QuickStartListCard>().apply {
+            add(
+                    QuickStartHeaderCard(
+                            title = UiStringRes(headerTitleResId),
+                            shouldShowHeaderImage = !displayUtilsWrapper.isPhoneLandscape()
+                    )
+            )
+            addAll(taskCards)
+        }.toList()
+        quickStartAdapter.submitList(quickStartList)
     }
```

Comment:
Suggestion: I think we can extract the lines 85-94 to a function which creates and returns a `QuickStartHeaderCard` and then add it to the list. WDYT 🤔 . Something like this 

```
fun getHeaderCard() {
        val headerTitleResId = quickStartCardBuilder.getTitle(tasksType)
        val header =  QuickStartHeaderCard(
                            title = UiStringRes(headerTitleResId),
                            shouldShowHeaderImage = !displayUtilsWrapper.isPhoneLandscape()
          )
       return header
}
```

Reply:
Good point.

Actually, this fragment needs to utilize `ViewModel` and Card-`Builder` pattern, hopefully, we'll get time to do that in near future. Till that time, included your suggestion in 8e6c6b0.

Reply:
Yep, Thanks for making the change. 👍🏼